pkg/build/api/types.go

```diff
@@ -55,6 +55,9 @@ type BuildParameters struct {
 
 	// Output describes the Docker image the Strategy should produce.
 	Output BuildOutput `json:"output,omitempty"`
+
+	// Compute resource requirements to execute the build
+	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
I can make it a pointer, but you would be inconsistent with upstream:

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/types.go#L292
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/types.go#L595

Typically, I prefer to just stay consistent so folks know if they call .Resources if they have a pointer or not.